dropwizard-views-mustache/src/test/java/io/dropwizard/views/mustache/MustacheViewRendererTest.java

```diff
@@ -73,12 +80,32 @@ public void rendersViewsWithAbsoluteTemplatePaths() throws Exception {
     }
 
     @Test
+    public void rendersViewsWithAbsoluteTemplatePathsFromFileSystem() throws Exception {
```

Comment:
I think it would be better to create an additional independent test class (which starts a new server, application, etc...) that verifies loading templates from the filesystem, rather than reusing the same renderer across different tests.